From e21596935f984a286b11f2068e15ff66166ffe7b Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Tue, 12 May 2020 10:15:51 +0700
Subject: [PATCH] arm64: dts: renesas; r8a774e1-hihope-rzg2h: Reduce memory
 when enable ECC full mode.

When enable ecc full mode, the half of memory is used for ECC area.
We must reduce it to avoid issue happen when user access to.
When enable ecc full mode, DRAM won't be in split mode, so the address
offset of 2nd DRAM is 0x6.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
index 30b83d58c318..2551fdf86e9a 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
@@ -16,12 +16,12 @@
 	memory@48000000 {
 		device_type = "memory";
 		/* first 128MB is reserved for secure area. */
-		reg = <0x0 0x48000000 0x0 0x78000000>;
+		reg = <0x0 0x48000000 0x0 0x4C000000>;
 	};
 
 	memory@600000000 {
 		device_type = "memory";
-		reg = <0x6 0x00000000 0x0 0x80000000>;
+		reg = <0x6 0x00000000 0x0 0x40000000>;
 	};
 
 	reserved-memory {
-- 
2.17.1

